src/Psalm/Codebase.php

```diff
@@ -299,8 +300,8 @@ final class Codebase
      */
     public $allow_backwards_incompatible_changes = true;
 
-    /** @var int */
-    public $analysis_php_version_id = PHP_VERSION_ID;
+    /** @var int<50400,80299> */
+    public $analysis_php_version_id;
```

Comment:
Off-topic, but should it be a space after the comma? Not sure phpcs supports that